pymongoexplain/commands.py

```diff
@@ -110,18 +122,30 @@ def __init__(self, collection: Collection,
         super().__init__(collection.name)
         for key, value in kwargs.items():
             self.command_document[key] = value
+        if self.command_document["filter"] == {}:
+            self.command_document = {}
```

Comment:
Is this correct? What about all the other find command options? Like:
```
coll.find({}, limit=10)
```

Reply:
This was the only way to make all the tests pass, so I'm not sure.

Reply:
Please add a test for `coll.find({}, limit=10)` to ensure that the limit field (and other command args) are not removed.

Reply:
Added a test.